packages/frontend/src/components/ElectionForm/Details/ElectionAuthForm.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react'
+import { FormControl, FormControlLabel, FormLabel, Grid, Paper, Radio, Typography } from "@mui/material"
+import useElection from '../../ElectionContextProvider';
+import { useSubstitutedTranslation } from '~/components/util';
+
+export default function ElectionAuthForm() {
+
+    const { election, refreshElection, permissions, updateElection } = useElection()
+    const ip = election.settings.voter_authentication.ip_address == true
+    const device_id = election.settings.voter_authentication.voter_id == true
+    const email = election.settings.voter_authentication.email == true
+    const none = !(ip || device_id || email)
+
+    const handleUpdate = async (voter_authentication) => {
+        await updateElection(election => election.settings.voter_authentication = voter_authentication)
+        await refreshElection()
+    }
+
+    
+    const {t} = useSubstitutedTranslation('election');
+
+    return (
+        <Paper elevation={3}>
+            <Grid container
+                sx={{
+                    m: 0,
+                    p: 1,
+                }}
+            >
+                <Grid xs={12}>
+                    <Typography gutterBottom variant="h4" component="h4">
+                        {t('admin_home.voter_authentication.form_label')}
+                    </Typography>
+                </Grid>
+                <FormControl>
+                    <FormLabel id="demo-radio-buttons-group-label">{t('admin_home.voter_authentication.help_text')}</FormLabel>
+                    <FormControlLabel value="female" control={
```

Comment:
Was value="female" intentional? Maybe it was copied from somewhere?

Reply:
Unintentional, copied from MUI example.